login-workflow/docs/existing-project-migration.md

```diff
@@ -0,0 +1,135 @@
+# Update React Auth Workflow Package
+
+Update @brightlayer-ui/react-auth-workflow package to latest version.
+```shell
+npm install --save @brightlayer-ui/react-auth-workflow
+// or
+yarn upgrade @brightlayer-ui/react-auth-workflow
+```
+
+## Remove @brightlayer-ui/react-auth-shared package
+
+In latest version we have removed @brightlayer-ui/react-auth-shared.
+
+## Update AuthUIActions
+
+Since we have removed `@brightlayer-ui/react-auth-shared` package `SecurityContextActions` are not supported by latest workflow. Instead of this you need to manage your app's state by app context. 
+
+For e.g. you can replace SecurityContextActions with `AppContextType` which you can declare in AppContextProvider. You can check [here](https://github.com/etn-ccis/blui-react-workflows/blob/master/login-workflow/example/src/contexts/AppContextProvider.tsx) for detailed implementation.
+
+In previous package `onUserAuthenticated`, `onUserNotAuthenticated` were managed by `@brightlayer-ui/react-auth-shared` but in latest you need to manage through app state. For more details you can check Example app's [AuthUIActions Implementation](https://github.com/etn-ccis/blui-react-workflows/blob/master/login-workflow/example/src/actions/AuthUIActions.tsx)
+
+## Update RegistrationUIActions
+
+Update type name `AccountDetailInformation` to `AccountDetails`.
+
+// before
+```tsx
+import { RegistrationUIActions, AccountDetailInformation } from '@brightlayer-ui/react-auth-workflow';
+```
+
+// after
```

Comment:
For the before and after. Instead of using javascript comment syntax, you can just make them bold with markdown for nicer formatting. Do this everywhere before and after are used this way
```suggestion
**before:**
```tsx
import { RegistrationUIActions, AccountDetailInformation } from '@brightlayer-ui/react-auth-workflow';
```

**after:**
```